src/System.ServiceModel.Primitives/ref/System.ServiceModel.Primitives.cs

```diff
@@ -4,7 +4,12 @@
 // Changes to this file must follow the http://aka.ms/api-review process.
 // ------------------------------------------------------------------------------
 
-
+namespace System.IdentityModel.Selectors {
+  public abstract partial class X509CertificateValidator {
+    protected X509CertificateValidator() { }
+    public abstract void Validate(System.Security.Cryptography.X509Certificates.X509Certificate2 certificate);
+  }
+}
```

Comment:
If we ever needed to expose the rest of that type what do you guys think we would do? Would we add the rest of it into S.SM.Primitives contract? Or would we create another IdentityModel contract and have it be a dependency of S.SM.Primtives and type forward this type into there? I just want to make sure we think about what we would do with this in the future.